dist/index.d.ts

```diff
@@ -2174,12 +2174,15 @@ type FIND_RUINS = 123;
 
 // Filter Options
 
-interface FilterOptions<T extends FindConstant, S extends FindTypes[T] = FindTypes[T]> {
-    filter: FilterFunction<FindTypes[T], S> | FilterObject | string;
+interface PredicateFilterOptions<T, S extends T> {
```

Comment:
would anyone give this a better name?

Reply:
We could probably avoid it, which I would argue we should.
There is already a default for FilterOptions, so you just need to add an extra, optional, parameter to the `find*` methods

Reply:
`PredicateFilterOptions` has been merged into `FilterOption` now. But there is still a `PredicateFilterFunction`. Let's keep it as-is now. Feel free to give any better name.